testproject/testapp/tests.py

```diff
@@ -27,6 +29,9 @@ class RootViewTest(restframework.APIViewTestCase,
                    assertions.StatusCodeAssertionsMixin):
     view_class = djoser.views.RootView
 
+    def tearDown(self):
+        clear_url_caches()
```

Comment:
@mmarksnippety can you leave short comment in a code why it is here? I guess that django urls needs some internal clean-up after changing `ROOT_URLCONF` in `test_get_should_return_urls_mappings_whit_namespace_setup()` - am I right?